doc/advanced_configuration.rst

```diff
@@ -312,3 +312,21 @@ the example script.
 
 
 .. _regular expressions: https://docs.python.org/2/library/re.html
+
+
+Pandoc for converting rst to markdown for notebook downloads
+============================================================
+
+Sphinx uses restructured text while Jupyter notebooks uses markdown for
+markup of the text. Therefore, restructured text need to be translated to
```

Comment:
Ok, here's what I think we should write:

Sphinx uses restructured text while Jupyter notebooks uses markdown for formatting the text. Due to this mismatch in markup formats, the generated Jupyter notebooks may not display the text correctly if you have heavily used the formatting syntax of restructured text. 

Sphinx-Gallery provides a minimal parser to convert from restructured text to markdown. It is suitable for basic use cases and is the default parser. Sphinx-Gallery is nevertheless aware of [pypandoc](https://github.com/bebraw/pypandoc) and if it is found in your system, it will be used to conversion. To install `pypandoc` in your system you can use the following commands: